src/key-value-engine/rocksdb.md

```diff
@@ -0,0 +1,155 @@
+# RocksDB
+
+RocksDB is a persistent key-value store for fast storage environment.
```

Comment:
Would it be better like this?
```
RocksDB is a C++ library, which provides a persistent.....
```

Reply:
I actually copied that part from https://rocksdb.org, so I think it is fine :)